src/bin/pg_autoctl/Makefile

```diff
@@ -32,7 +32,8 @@ CFLAGS += -Wformat
 CFLAGS += $(COMMON_LIBS)
 
 LIBS  = -L $(shell $(PG_CONFIG) --libdir)
-LIBS += -L $(shell $(PG_CONFIG) --pkglibdir)
+LIBS  = -L $(shell $(PG_CONFIG) --pkglibdir)
```

Comment:
we should += to LIBS

Reply:
Good spot! It's actually the other way round, I left the --libdir line even though it's not needed anymore, as I just re-tested. Re-uploaded new code (as a rebase).